applications/openshift/api-server/ocp_api_server_audit_log_maxsize/rule.yml

```diff
@@ -0,0 +1,48 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: 'Configure OpenShift API Server Maximum Audit Log Size'
+
+description: |-
+    To rotate audit logs upon reaching a maximum size,
+    edit the <tt>openshift-apiserver</tt> configmap on
+    the master node(s) and set the <tt>audit-log-maxsize</tt> parameter to
+    an appropriate size in MB. For example, to set it to 100 MB:
+    <pre>
+    "apiServerArguments":{
+      ...
+      "audit-log-maxsize": ["100"],
+      ...
+    </pre>
+
+rationale: |-
+    OpenShift automatically rotates log files. Retaining old log files ensures that
+    OpenShift Operators have sufficient log data available for carrying out any
+    investigation or correlation. If you have set file size of 100 MB and the number of
+    old log files to keep as 10, there would be approximately 1 GB of log data
+    available for use in analysis.
+
+severity: medium
+
+references:
+    cis: 1.2.25
+
+ocil_clause: '<tt>audit-log-maxsize</tt> is set to <tt>100</tt> or as appropriate'
+
+ocil: |-
+    Run the following command on the master node(s):
+    <pre>$ oc get configmap config -n openshift-apiserver -ojson | jq -r '.data["config.yaml"]' | jq '.apiServerArguments["audit-log-maxsize"]'</pre>
+    The output should return a value of <pre>["100"]</pre> or as appropriate.
+
+template:
+  name: yamlfile_value
+  vars:
+    ocp_data: "true"
```

Comment:
With these you'll need the 'openshift_cluster_setting' warning section

Reply:
yep! I do... these fails happen to pass because other rules take the same file into use. But if I would run them individually they wouldn't get the configmap on to the scan. Thanks for catching this!